### DISCUSSION TITLE
Using zero-ui API

### DISCUSSION THREAD

**Discussion by MrSuicideParrot** [dec0dOS/zero-ui#11] [Q&A] [answered]
My objective is to use zero-ui as a replacement of the ZeroTier Central. 
But I don't understand what is the correct way to generate API tokens? 
I don't find a place to generate them in the UI, but I noticed  that they are used as authentication.
If I use one of these tokens for how long are they valid?

**Comment by dec0dOS** [comment, marked as answer]
Hello, @MrSuicideParrot!
For now, there is no way to manage it from the UI. For every user, a new token is created. You could get the token from the db.json file, it does not expire over time.

**Reply by MrSuicideParrot** [reply]
Thanks! This solves my problem.
